### PR TITLE
Added Schedules restrictions with  command who-s-on-call

### PR DESCRIPTION
**Pull request by eldios**
Added PagerDuty rescrition option (basic feature) to restrict the `who's on call` command.

Re: https://github.com/hubot-scripts/hubot-pager-me/issues/109#issuecomment-440433835

### CODE REVIEW

**Comment by eldios** [comment]
@stephenyeargin README.md updated: :+1:

File: src/scripts/pagerduty.coffee [hunk 1]
```diff
@@ -607,10 +608,12 @@ module.exports = (robot) ->
     scheduleName = msg.match[3] or msg.match[4]
 
     messages = []
+    allowed_schedules = pagerDutySchedules.split(",")
     renderSchedule = (s, cb) ->
       withCurrentOncall msg, s, (username, schedule) ->
         if (username)
-          messages.push("* #{username} is on call for #{schedule.name} - #{schedule.html_url}")
+          if !allowed_schedules or (allowed_schedules and schedule.id in allowed_schedules)
```
[the remark this answers is not in the document]
Comment (eldios) [reply src/scripts/pagerduty.coffee]: added

[the remark this answers is not in the document]
Comment (eldios) [reply src/scripts/pagerduty.coffee]: @a-palchikov is that okay now?

**Comment by stephenyeargin** [comment]
Released with [`v3.1.0`](https://github.com/hubot-scripts/hubot-pager-me/releases/tag/v3.1.0)

**Pull request merged**